crates/weaver_semconv/src/any_value.rs

```diff
@@ -0,0 +1,690 @@
+                    }
+                },
+                AnyValueSpec::String {
+                    common: AnyValueCommonSpec {
+                        id: "id_string".to_owned(),
+                        brief: "brief".to_owned(),
+                        note: "note".to_owned(),
+                        stability: None,
+                        examples: None,
+                        requirement_level: RequirementLevel::Basic(BasicRequirementLevelSpec::Recommended),
+                        deprecated: None,
+                    }
+                },
+                AnyValueSpec::Boolean {
+                    common: AnyValueCommonSpec {
+                        id: "id_bool".to_owned(),
+                        brief: "brief".to_owned(),
+                        note: "note".to_owned(),
+                        stability: None,
+                        examples: None,
+                        requirement_level: RequirementLevel::Basic(BasicRequirementLevelSpec::Optional),
+                        deprecated: None,
+                    }
+                },
+                AnyValueSpec::Double {
+                    common: AnyValueCommonSpec {
+                        id: "id_double".to_owned(),
+                        brief: "brief".to_owned(),
+                        note: "note".to_owned(),
+                        stability: None,
+                        examples: None,
+                        requirement_level: RequirementLevel::Basic(BasicRequirementLevelSpec::Optional),
+                        deprecated: None,
+                    }
+                },
+                AnyValueSpec::Doubles {
+                    common: AnyValueCommonSpec {
+                        id: "id_doubles".to_owned(),
+                        brief: "brief".to_owned(),
+                        note: "note".to_owned(),
+                        stability: None,
+                        examples: None,
+                        requirement_level: RequirementLevel::Basic(BasicRequirementLevelSpec::Optional),
+                        deprecated: None,
+                    }
+                },
+            ],
+        };
+        let body = BodySpec {
+            body: map.clone(),
+        };
+
+        let expected_yaml = fs::read_to_string("data/expected/any_value.yaml").unwrap().replace("\r\n", "\n");
+        assert_eq!(expected_yaml, format!("{}", serde_yaml::to_string(&body).unwrap()), "{}", expected_yaml);
+
+        let expected_json = fs::read_to_string("data/expected/any_value.json").unwrap().replace("\r\n", "\n");
+        assert_eq!(expected_json, format!("{}", serde_json::to_string(&body).unwrap()), "{}", expected_json);
+
+        assert_eq!(
+            format!(
+                "{}",
+                map
+            ),
+            "map<id>{ enum<id_enum>, map<id_map>{ int, byte[], string, boolean, map<id_nested_map>{ int[], double[], string[], boolean[] } }, int, byte[], string, boolean, double, double[] }"
```

Comment:
Discussion Point:
This is the "example" of the Display from using the names above.
I went through a few iterations (one included the `id:type` for each "field" (but it made it very long), as the id is mandatory.

eg.
`map<id>{ enum<id_enum>, map<id_map>{ id_int:int, id_bytes:byte[], ...`